### ISSUE TITLE
Unable to iterate over services

### ISSUE THREAD

**Issue by naveenkumarmarri** [periscope-ps/UNISrt#22]
Branch: `develop`

Unable to iterate over the services. Following is the piece of code, I tried

```python
for service in rt.services:
    print(service)
```

**Comment by jemusser** [comment]
Could you include a stack trace for this error?

**Comment by naveenkumarmarri** [comment]
I've few service resources added to the runtime, But nothing is returning from the above piece of code.

**Comment by disprosium8** [comment]
```>>> rt = unis.Runtime([{"default": True, "url": "http://ezra-x1:9000"}, {"url": "http://172.19.0.2:9000"}])
>>> 
>>> rt.services
<UnisList.services [...]>
>>> for s in rt.services:
...  print(s.id)
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ezra/repos/unisrt/unis/models/lists.py", line 31, in __iter__
    for v in self._obj.__iter__():
  File "/home/ezra/repos/lace/lace/logging/_log.py", line 235, in wrapper
    return f(*args, **kwargs)
  File "/home/ezra/repos/unisrt/unis/models/lists.py", line 295, in __iter__
    self._complete_cache()
  File "/home/ezra/repos/lace/lace/logging/_log.py", line 235, in wrapper
    return f(*args, **kwargs)
  File "/home/ezra/repos/unisrt/unis/models/lists.py", line 210, in _proto_complete_cache
    self._get_next()
  File "/home/ezra/repos/lace/lace/logging/_log.py", line 235, in wrapper
    return f(*args, **kwargs)
  File "/home/ezra/repos/unisrt/unis/models/lists.py", line 230, in _proto_get_next
    self.append(model(result))
  File "/home/ezra/repos/unisrt/unis/models/models.py", line 358, in __call__
    instance = super(_jsonMeta, cls).__call__(*args, **kwargs)
  File "/home/ezra/repos/unisrt/unis/models/models.py", line 358, in __call__
    instance = super(_jsonMeta, cls).__call__(*args, **kwargs)
  File "/home/ezra/repos/unisrt/unis/models/models.py", line 358, in __call__
    instance = super(_jsonMeta, cls).__call__(*args, **kwargs)
  File "/home/ezra/repos/lace/lace/logging/_log.py", line 235, in wrapper
    return f(*args, **kwargs)
  File "/home/ezra/repos/unisrt/unis/models/models.py", line 252, in __init__
    self.__dict__.update({**self._rt_defaults, **{k:self._lift(x, k, None) for k,x in v.items()}})
  File "/home/ezra/repos/unisrt/unis/models/models.py", line 252, in <dictcomp>
    self.__dict__.update({**self._rt_defaults, **{k:self._lift(x, k, None) for k,x in v.items()}})
  File "/home/ezra/repos/lace/lace/logging/_log.py", line 235, in wrapper
    return f(*args, **kwargs)
  File "/home/ezra/repos/unisrt/unis/models/models.py", line 125, in _lift
    v = ctx.insert(v) if "$schema" in v else ctx.find(v['href'])[0]
AttributeError: 'NoneType' object has no attribute 'find'
>>> ```

**Comment by jemusser** [comment]
@disprosium8 This is actually a separate issue, but there should be an issue for it as well.  @naveenkumarmarri is working on the multisource branch and has been having a different issue with services, which I want to check for in my new implementation to make sure it does not re-emerge.

**Comment by naveenkumarmarri** [comment]
No, I'm working on `develop` branch,I've mentioned in the issue description.

When I iterate over `service.id` I'm getting same error as @disprosium8 has posted

**Comment by jemusser** [comment]
I stand corrected, I thought this was the issue previously discussed.

**Comment by jemusser** [comment]
This should have been resolved with issue/#20, closing for now.  If this appears in a different context I will reopen.
